include/deal.II/matrix_free/matrix_free.templates.h

```diff
@@ -1315,17 +1317,22 @@ MatrixFree<dim, Number, VectorizedArrayType>::initialize_indices(
   Table<2, internal::MatrixFreeFunctions::ShapeInfo<double>> shape_info_dummy(
     shape_info.size(0), shape_info.size(2));
   {
-    QGauss<dim> quad(1);
+    Quadrature<dim> quad(QGauss<dim>(1));
+    Quadrature<dim> quad_simplex(Simplex::QGauss<dim>(1));
```

Comment:
It is not great that here a dummy quadrature rule of type `QGauss` is used. Any ideas?